packages/config/eslint-preset.js

```diff
@@ -19,6 +19,18 @@ module.exports = {
     },
   },
   rules: {
+    "@typescript-eslint/no-restricted-imports": [
+      "error",
+      {
+        paths: [
+          {
+            name: "@prisma/client",
+            message: "Import from @calcom/prisma (or @calcom/prisma/enums) instead",
+            allowTypeImports: true,
+          },
+        ],
+      },
+    ],
```

Comment:
added this just to make sure there we're no more anomalies left - but might be good to keep it so that you don't import runtime stuff from here in the future